OrbitService/CaptureServiceImpl.cpp

```diff
@@ -91,30 +91,52 @@ class GrpcCaptureEventSender final : public CaptureEventSender {
     CHECK(reader_writer_ != nullptr);
   }
 
+  ~GrpcCaptureEventSender() {
+    LOG("Total number of events sent: %lu", total_number_of_events_sent_);
+    LOG("Total number of bytes sent: %lu", total_number_of_bytes_sent_);
+    float average_bytes =
+        total_number_of_events_sent_ == 0
+            ? 0.0f
+            : static_cast<float>(total_number_of_bytes_sent_) / total_number_of_events_sent_;
```

Comment:
Dividing a float by zero doesn't produce any exception, it just gives NaN (technically more correct than 0...).

Reply:
(Well, NaN if the dividend is also 0, plus or minus infinity otherwise.)

Reply:
Ah, of course. Was thinking of integer division for a moment. Reverted the change (which is also more concise). Thanks a lot!